include/sdf/JointAxis.hh

```diff
@@ -69,6 +77,18 @@ namespace sdf
     public: [[nodiscard]] sdf::Errors SetXyz(
                 const gz::math::Vector3d &_xyz);
 
+    /// \brief Set the joint to be mimicked.
+    /// \param[in] _mimicJoint The joint name, multiplier, and offset to be used
+    /// for mimicking.
+    /// \sa mimicJoint MimicJoint()
+    public: void SetMimicJoint(const mimicJoint _mimicJoint);
+
+    /// \brief Get the joint to be mimicked.
+    /// \return The joint name, multipler, and offset of the joint to
+    /// be mimicked.
+    /// \sa void SetMimicJoint(const mimicJoint)
+    public: mimicJoint MimicJoint() const;
```

Comment:
should this return a `std::optional<mimicJoint>` in case one is not set?

Reply:
I think it should return a `std::optional<...>`

Reply:
Done https://github.com/gazebosim/sdformat/pull/1166/commits/573d6fd0da4360a9b93b3a1219c23e08d8b096b8